### PR TITLE
Added Cross Ange, Girls Bravo, Juden Chan, KanColle: Kantai Collectio…

### PR DESCRIPTION
**Pull request by DexLuther**
…n and Million Arthur

- Added Cross Ange: https://anilist.co/anime/20806/Cross-Ange-Rondo-of-Angel-and-Dragon/
- Added Girls Bravo: 
    - Season 1: https://anilist.co/anime/241/Girls-Bravo/  
    - Season 2: https://anilist.co/anime/487/Girls-Bravo-Second-Season/
- Added Juden Chan: https://anilist.co/anime/5973/Juden-Chan/
- Added KanColle: Kantai Collection:  
  - Season 1: https://anilist.co/anime/20553/KanColle/ 
  - Season 2: https://anilist.co/anime/21294/KanColle-Season-2-Lets-Meet-at-Sea/
- Added Million Arthur:
   - Season 1: https://anilist.co/anime/101359/Million-Arthur/
   - Season 2: https://anilist.co/anime/104071/Million-Arthur-2/

### CODE REVIEW

File: custom_mappings.yaml [hunk 1]
```diff
@@ -499,6 +499,12 @@ entries:
     seasons:
       - season: 1
         anilist-id: 15037
+  - title: "Cross Ange: Rondo of Angel and Dragon"
+    synonyms:
+      - "Cross Ange: Rondo of Angels and Dragons"
```
Comment (mizz141) [inline custom_mappings.yaml:504]: add more synonyms:

`- "Cross Ange: Tenshi to Ryuu no Rondo"`

File: custom_mappings.yaml [hunk 2]
```diff
@@ -1276,6 +1288,14 @@ entries:
         anilist-id: 98314
       - season: 2
         anilist-id: 100876
+  - title: "KanColle"
+    synonyms:
+      - "KanColle: Kantai Collection"
```
Comment (mizz141) [inline custom_mappings.yaml:1293]:
```
    synonyms:
      - "KanColle: Let's Meet at Sea" 
      - "KanColle - Fleet Girls Collection -"
      - "Kantai Collection: KanColle"
```

File: custom_mappings.yaml [hunk 3]
```diff
@@ -2040,6 +2060,12 @@ entries:
      - season: 2
        anilist-id: 102941
        start: 11
+  - title: "Juden Chan"
+    synonyms:
+      - "Charger Girl Juden Chan"
```
Comment (mizz141) [inline custom_mappings.yaml:2065]:
```
    synonyms:
      - "Fight Ippatsu! Juuden-chan!!"
      - "Fight Ippatsu! Jūden-chan!!"
      - "Fight, One Shot! Charger Girls!!"
      - "Charger Girl Ju-den Chan"
```

File: custom_mappings.yaml [hunk 4]
```diff
@@ -2048,6 +2074,14 @@ entries:
     seasons:
       - season: 1
         anilist-id: 11179 
+  - title: "Million Arthur"
+    synonyms:
+      - "Operation Han-Gyaku-Sei Million Arthur"
```
Comment (mizz141) [inline custom_mappings.yaml:2079]:
```
    synonyms:
      - "Han-Gyaku-Sei Million Arthur"
      - "Hangyakusei Million Arthur"
```

**Review by mizz141 — CHANGES_REQUESTED** [review]

**Comment by mizz141** [comment]
Hello, thank you for taking your time to contribute to this project, I've requested some additional synonyms to be added before I can go ahead and merge your PR.

**Comment by DexLuther** [comment]
> Hello, thank you for taking your time to contribute to this project, I've requested some additional synonyms to be added before I can go ahead and merge your PR.

I think I updated it. Not really used to using Github, so not sure if I did it right or if it worked. lol

**Review by mizz141 — APPROVED** [review]

**Comment by mizz141** [comment]
> > Hello, thank you for taking your time to contribute to this project, I've requested some additional synonyms to be added before I can go ahead and merge your PR.
> 
> I think I updated it. Not really used to using Github, so not sure if I did it right or if it worked. lol

The github action ran through without issue, so you did everything right on that side, I'll merge the commit and it should be live next time you run the script.

**Comment by mizz141** [comment]
Also you might want to look into your github settings and change your commit address since it's leaking your email: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address

**Pull request merged**